lib/config-seed/nginx.conf

```diff
@@ -1,6 +1,11 @@
 events {}
 
 http {
+    resolver 127.0.0.11 ipv6=off valid=10s;
```

Comment:
Suggest adding a comment that 127.0.0.11 is the address of the docker resolver (as seen inside the docker containers)